cmd/v/help/symlink.txt

```diff
@@ -0,0 +1,5 @@
+Usage: v symlink
+
+This command adds a symlink for the V compiler executable.
+
+Note that on Unix systems this command requires root to work properly.
```

Comment:
It does not directly - it requires a write permissions to the `/usr/local/bin` folder, where it will put the symlink.
On some unix systems (older macos + brew), that folder is writable by the user that installed brew.